### DISCUSSION TITLE
Pb with downloading file in the working directory

### DISCUSSION THREAD

**Discussion by laguinch** [photobooth-app/photobooth-app#159] [General]
Hy, i have a problem with the files in the files directory . I have the warning icon and loading and nothing is coming ? Thx

**Comment by mgineer85** [comment]
You mean downloading in admin backend filebrowser or download from gallery? 
any logs?
Which warning icon you refer to?
Screenshots?

**Reply by laguinch** [reply]
Yes in the admin backend. I can't test any more because I reinstall the Photo Booth app sorry

**Reply by mgineer85** [reply]
Ok I will test later on my installation if there is something wrong.
You mean after the update it’s fine?

**Reply by laguinch** [reply]
Now it's work but I have much folders 
<img width="1413" alt="Capture d’écran 2024-01-14 à 14 13 48" src="https://github.com/photobooth-app/photobooth-app/assets/63105209/a86a14c7-f89f-4399-b5dc-037da5024534">

**Reply by laguinch** [reply]
I change the path in the ~/.local/share/systemd/user/photobooth-app.service file and it's ok
<img width="1430" alt="Capture d’écran 2024-01-14 à 14 19 33" src="https://github.com/photobooth-app/photobooth-app/assets/63105209/f8490399-afc8-4620-ae41-eebf01f630ad">

**Reply by laguinch** [reply]
One more question, what is the technic to lunch the Photo Booth-app.desktop on the rpi screen ? I don't know how to double clic on the screen ?
Thx

**Reply by mgineer85** [reply]
That’s because the current directory you are in will be used as working directory.

**Reply by mgineer85** [reply]
Actually there is no special technique. You just need to doubleclick to make it start.

**Reply by laguinch** [reply]
it doesn't work on my screen :( the double tap with my finger don't open the app. But with the mouse it's ok.

**Reply by mgineer85** [reply]
Maybe you find an answer from the system manufacturer or there is an issue with you screen? 

Regarding the download again, maybe you tried to download a very huge amount of files?

**Reply by laguinch** [reply]
Perhaps, i don’t know.
